packages/quantic/cypress/e2e/default-2/recommendation-list/recommendation-list-selectors.ts

```diff
@@ -30,7 +30,7 @@ export const RecommendationListSelectors: RecommendationListSelector &
     ),
   recommendationLinks: () =>
     RecommendationListSelectors.get().find(
-      '[data-cy="recommendations__item"] c-quantic-result-link a'
+      '[data-cy="recommendations__item"] [data-cy="result-link"] a'
```

Comment:
Why exactly are we changing this?

Reply:
just because the selectors where not precise enough,  previously we  were selecting  `c-quantic-result-link`  and our goal was only to select the links that are used to display the title of the result.

However now, since `c-quantic-result-link` is now used by the `QuanticResultPrintableUri` it means that we were selecting some links that we weren't supposed to select.

So updated the selectors to be more precise, yielding the selection only of the result titles.

Reply:
Yeah according to the cypress documentation, `data-cy` seems like the best option

Reply:
makes sense 👍